audio-worklet/basic/js/porter.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * A simple MessagePort tester.
+ * 
+ * @class PorterProcessor
+ * @extends AudioWorkletProcessor
+ */
+class PorterProcessor extends AudioWorkletProcessor {
+  constructor() {
+    super();
+    this._lastUpdate = currentTime;
+    this.port.onmessage = this.handleMessage.bind(this);
+  }
+
+  handleMessage(event) {
+    console.log('[Processor:Received] "' + event.data.message + 
+                '" (' + event.data.timeStamp + ')');
+  }
+
+  process() {
+    if (currentTime - this._lastUpdate > 1.0) {
```

Comment:
Add comment on why you're doing this test.

Reply:
Done.